RazorGenerator.Core/DirectivesParser.cs

```diff
@@ -62,7 +62,8 @@ private static void ParseFileDirectives(Dictionary<string, string> directives, s
         private static void ParseKeyValueDirectives(Dictionary<string, string> directives, string directivesLine)
         {
             // TODO: Make this better.
-            var regex = new Regex(@"\b(?<Key>\w+)\s*:\s*(?<Value>[~\\\/\w\.]+)\b", RegexOptions.ExplicitCapture);
+            const string valueRegexPattern = @"[~\\\/\w\.]+";
+            var regex = new Regex(@"\b(?<Key>\w+)\s*:\s*(?<Value>" + valueRegexPattern + @"(\s*,\s*" + valueRegexPattern + @")*)\b", RegexOptions.ExplicitCapture);

```

Comment:
That's probably been there forever. Still, a simple comment with an example of what that parses can't hurt. e.g.

    SomeKey: foo SomeOtherKey: bar1, bar2